gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/datafetcher/EntityDataServiceEntityFetcher.java

```diff
@@ -212,4 +202,23 @@ public long getTotal(EntitiesRequestContext requestContext, EntitiesRequest enti
         entityQueryServiceClient.total(totalEntitiesRequest, requestContext.getHeaders());
     return response.getTotal();
   }
+
+  private Map<String, AttributeMetadata> getAttributeMetadataByAlias(
+      EntitiesRequestContext requestContext, EntitiesRequest request) {
+    Map<String, AttributeMetadata> attributeMetadataByIdMap =
+        attributeMetadataProvider.getAttributesMetadata(requestContext, request.getEntityType());
+    return request.getSelectionList().stream()
+        .filter(
+            expression ->
+                ExpressionReader.getSelectionAttributeId(expression)
+                    .map(attributeMetadataByIdMap::containsKey)
+                    .orElse(false))
+        .map(
+            expression ->
+                Map.entry(
+                    ExpressionReader.getSelectionResultName(expression).orElseThrow(),
```

Comment:
Can we rename this as `ExpressionReader.getSelectionAttributeId` -> `ExpressionReader.getAttributeId`?

Reply:
Started discussion in https://github.com/hypertrace/gateway-service/pull/116/files/ea0965d95e98c7caaa83dea5501dc17437362189#r778870420